backend/src/main/java/com/google/sps/notifiers/EmailNotifier.java

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main.java.com.google.sps.notifiers;
+
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.*;
+
+import java.io.IOException;
+
+/** An instance of the email notifier class is used to send emails to participants. */
+public class EmailNotifier extends Notifier {
+  private String toEmail;
+
+  /**
+   * @param name Name of the recipient of the user
+   * @param toEmail1 main.java.com.google.sps.Email that this email is going to sent to.
+   */
+  public EmailNotifier(String name, String toEmail1) {
+    super(name);
+    this.toEmail = toEmail1;
+    setToEmail(toEmail);
```

Comment:
I'd recommend not calling any instance methods from a constructor. Calling methods from a constructor requires that those methods can handle cases where `this` isn't setup yet. Take for example a class that has several final fields that are assigned in the constructor. If the constructor calls a method before assigning those fields, that method can no longer assume that those fields are initialized even though they're final.

Reply:
Ack